test/vcpkg-configuration.json

```diff
@@ -0,0 +1,22 @@
+{
+  "registries": [
+    {
+      "kind": "artifact",
+      "location": "https://aka.ms/vcpkg-ce-default",
+      "name": "microsoft"
+    },
+    {
+      "kind": "artifact",
+      "location": "https://artifacts.tools.arm.com/vcpkg-ce-registry/registry.zip",
+      "name": "arm"
+    }
+  ],
+  "requires": {
+    "microsoft:cmake": "^3.25.2",
+    "microsoft:ninja": "^1.10.2",
+    "arm:compilers/arm/armclang":"^6.21.0",
+    "arm:compilers/arm/arm-none-eabi-gcc": "^13.2.1",
+    "arm:tools/open-cmsis-pack/cmsis-toolbox": "^2.3.0",

```

Comment:
Important point to be discussed as a next step: while it's useful to periodically run the CI tests with the latest released toolbox to ensure the infrastructure and the tests themselves are up and running, it is also imperative to test the current state of each tool, for example by building their main branches or scheduling nightly releases. (FYI @jkrech)

Reply:
@brondani , Thank you for your valuable input.
Incorporating this would be a beneficial addition to these tests.

As we discussed, I prefer to implement this running strategy in a separate PR. This will help keep the changes clean and isolated